bindings/ruby/testsuite/function-call.rb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/ruby
+
+require 'test/unit'
+require 'storage'
+
+
+class TestFunctionCall < Test::Unit::TestCase
+
+  def test_function_call
+
+    environment = Storage::Environment.new(true, Storage::ProbeMode_NONE, Storage::TargetMode_DIRECT)
+    storage = Storage::Storage.new(environment)
+
+    assert_equal(storage.default_mount_by(), Storage::MountByType_UUID)
+    assert_raises(ArgumentError) { storage.default_mount_by("extra parameter") }
+    assert_raises(NoMethodError) { storage.does_not_exist() }
+
+    assert_equal(storage.public_send(:default_mount_by), Storage::MountByType_UUID)
```

Comment:
Why is it necessary to test the same both: calling directly and using `public_send`? Is there any difference?